### DISCUSSION TITLE
Local Notification localised permission message

### DISCUSSION THREAD

**Discussion by nabeelkhot** [ionic-team/capacitor#4263] [Q&A]
Hi,

I'm developing an app in Japanese and need the permission message for Local Notifications in Japanese.

How to achieve it?

**Comment by jcesarmobile** [comment]
To localize system UI messages such as permission prompts you have to add the language as supported by your app, check [official Apple docs](https://help.apple.com/xcode/mac/current/#/devafa3a605f)
